local.example.yml

```diff
@@ -38,7 +38,7 @@ sources:
     type: http
     url: https://msdl.microsoft.com/download/symbols/
     layout: { type: "symstore" }
```

Comment:
tbh, I have no idea why this is `symstore` and not `ssqp`. The only difference seems to be the casing, and the upstream microsoft server seems to be case insensitive anyway as far as I have tried.